topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+    They are also used along with the 'ChIP-seq' samples to identify the potential sequencing bias and help for differential analysis.
+
+
+> ### Agenda
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Step 1: Quality control and treatment of the sequences
+
+The first step of any ChIP-Seq data analysis is quality control of the raw sequencing data. 
+
+For the saving time, we will do it only on the data of one sample `wt_H3K4me3_rep1` that has been down-sampled.
+
+> ### {% icon hands_on %} Hands-on: Import the data
+>
+> 1. Create a new history for this tutorial and give it a proper name
+> 2. Import `wt_H3K4me3_read1.fastq.gz` and `wt_H3K4me3_read2.fastq.gz` from [Zenodo](https://zenodo.org/record/1324070) from the data library (ask your instructor)
+>
+>    ```
+>    https://zenodo.org/record/1324070/files/wt_H3K4me3_read1.fastq.gz
+>    https://zenodo.org/record/1324070/files/wt_H3K4me3_read2.fastq.gz
+>    ```
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > * Paste the link into the text field
+>    > * Press **Start** and **Close** the window
+>    > * Click on the `pencil` icon once the file is imported
+>    > * If format is not `fastqsanger`:
+>    > > * Click on **Datatype** in the middle panel
+>    > > * Select `fastqsanger` as **New Type**
+>    {: .tip}
+>
+>    > ### {% icon tip %} Tip: Importing data from a data library
+>    >
+>    > * Go into "Shared data" (top panel) then "Data libraries"
+>    > * Click on "Training data" and then "Analyses of ChIP-Seq data"
+>    > * Select interesting files
+>    > * Click on "Import selected datasets into history"
+>    > * Import in the history
+>    {: .tip}
+>
+>    As default, Galaxy takes the link as name, so rename them.
+>
+> 4. Rename the files `wt_H3K4me3_read1` and `wt_H3K4me3_read2`
+> 3. Inspect the first file by clicking on the {% icon galaxy-eye %} (eye) icon (**View data**)
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. How are the DNA sequences stored?
+>    > 2. What are the other entries?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > > 1. The DNA sequences are stored in the second line of every 4-line group
+>    > > 2. This file is called a [FastQ file](https://en.wikipedia.org/wiki/FASTQ_format). It stores sequence information and quality information. Each sequence is represented by a group of 4 lines with the 1st line being the sequence id, the second the sequence of nucleotides, the third a transition line and the last one a sequence of quality score for each nucleotide.
+>    > {: .solution }
+>    {: .question}
+>
+{: .hands_on}
+
+During sequencing, errors are introduced, such as incorrect nucleotides being called. These are due to the technical limitations of each sequencing platform. Sequencing errors might bias the analysis, ultimately leading to a misinterpretation of the data.
```

Comment:
Sequencing errors might bias an analysis and can lead to a misinterpretation of the data.